### DISCUSSION TITLE
Create a Field template picker similar to tag picker in the pop up window of the tiddlyclip add on

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#117] [Ideas]
Currently there is tag picker option in the tiddlyclip add on's pop up window. Is it possible to extend this functionality to include a field (template) picker. I meant if there are different template tiddlers with different set of custom fields, in a similar manner to picking tags, I should be able select the template tiddler with different set of fields for the clipped tiddler.

I meant to create multiple template tiddlers like $:/bj/tc/snipmetadata with different set of custom fields. Can it be possible to select the template tiddler from the pop up window of the addon similar to the tag picker

**Comment by buggyj** [comment]
It is not possible to do this.
